training/pyproject.toml

```diff
@@ -12,8 +12,6 @@ django = "^4.2.3"
 django-ninja = "^0.22.2"
 scikit-learn = "^1.3.0"
 pandas = "^2.0.3"
-torch = ">=2.0.0, !=2.0.1"
-torchvision = "^0.15"
```

Comment:
What version of `torch` and `torchvision` end up getting installed with this new config? @dwu359

Reply:
On my end, it installed `pytorch 2.0.1` and `torchvision 0.15.2` from the pytorch conda channel. This is specified in environment.yml:
  - pytorch::pytorch
  - pytorch::torchvision